internal/status/services_api_status.go

```diff
@@ -302,13 +302,13 @@ func UpdateSvcApiGatewayStatusObject(gw *svcapiv1alpha1.Gateway, updateStatus *s
 		updatedGW, err := lib.GetServicesAPIClientset().NetworkingV1alpha1().Gateways(gw.Namespace).Get(context.TODO(), gw.Name, metav1.GetOptions{})
 		if err != nil {
 			utils.AviLog.Warnf("gateway not found %v", err)
-			return err
```

Comment:
Don't we need the error anymore ?

Reply:
We've not been using those errors anywhere. We also don't have any logic to react to any these errors. In case of any error in such status update functions, we're just retrying the function. After a certain number of retries we eventually give that up as well.

Reply:
Makes sense. No need to return unnecessary errors.